packages/types/src/organizationMembership.ts

```diff
@@ -40,9 +36,18 @@ export interface OrganizationMembershipResource extends ClerkResource {
   update: (updateParams: UpdateOrganizationMembershipParams) => Promise<OrganizationMembershipResource>;
 }
 
+export type OrganizationCustomPermissionKey = string;
+export type OrganizationCustomRoleKey = string;
+
+/**
+ * @deprecated This type is deprecated and will be removed in the next major release.
+ * Use `OrganizationCustomRole` instead.
+ * MembershipRole includes `admin`, `basic_member`, `guest_member`. With the introduction of "Custom roles"
+ * these types will no longer match a developer's custom logic.
+ */
```

Comment:
❓ where is `OrganizationCustomRole` defined?

Reply:
Above, i just renamed it to `OrganizationCustomRoleKey`